wiki/Simultaneous_Release.md

```diff
@@ -17,14 +17,55 @@ existing simultaneous releases from the current and previous years.
 </tr>
 </thead>
 <tbody>
+
```

Comment:
Any reason we don't use [markdown tables](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-tables) instead of HTML? Your [original comment](https://github.com/eclipse-simrel/.github/issues/5#issuecomment-1747418764) states that the Planning Council decided to use MD